### PR TITLE
Radio button update

### PR DESCRIPTION
**Pull request by okolomiets**

### CODE REVIEW

File: src/directives/radiobutton/radiobutton.directive.ts [hunk 1]
```diff
@@ -7,20 +7,27 @@ export default () => {
...
+          value: '=ngValue'
         },
-        link: (scope: any, element: any, attrs: any, model: any) => {
-          model.$formatters.unshift(key => {
-              scope.isSelected = key === attrs.value;
-              return key;
-          });
+        link: {
+            post: (scope: any, element: any, attrs: any, model: any) => {
+                setTimeout(()=> {
```
Comment (kurpav) [inline src/directives/radiobutton/radiobutton.directive.ts]: What is this? Why do you need it?

File: src/directives/radiobutton/radiobutton.directive.ts [hunk 2]
```diff
@@ -7,20 +7,27 @@ export default () => {
         transclude: true,
         template: require('./radiobutton.partial.html'),
         scope: {
-            ngModel: '='
+          value: '=ngValue'
```
Comment (kurpav) [inline src/directives/radiobutton/radiobutton.directive.ts]: use one-way binding here

**Review by kurpav — CHANGES_REQUESTED** [review]

File: src/directives/radiobutton/radiobutton.directive.ts (hunk 2 above)
Comment (okolomiets) [reply src/directives/radiobutton/radiobutton.directive.ts]: fixed

File: src/directives/radiobutton/radiobutton.directive.ts (hunk 1 above)
Comment (okolomiets) [reply src/directives/radiobutton/radiobutton.directive.ts]: removed

**Review by freeman14 — APPROVED** [review]

**Review by kurpav — APPROVED** [review]

**Pull request merged**